bin/GameIndex.dbf

```diff
@@ -1804,15 +1817,15 @@ VU0KickstartHack = 1 // Fixes Character SPS.
 ---------------------------------------------
 Serial = SCES-51610
 Name   = This is Football 2004 (Red Devils 2004)
-Region = PAL-F-G
+Region = PAL-Du-F // Belgian.
```

Comment:
Lower down you capitalise DU, which is it? :P

Reply:
Guess i missed that one, i capitalised it now 👍